mne_bids/write.py

```diff
@@ -978,6 +980,14 @@ def write_anat(bids_root, subject, t1w, session=None, acquisition=None,
         also be a string pointing to a .trans file containing the
         transformation matrix. If None, no sidecar JSON file will be written
         for `t1w`
+    deface : tuple (float, float, str, bool) | None
```

Comment:
I would not deface by default. I would propose:

False : don't
True : do with (0.05, 35., 'erase', False) as defaults
tuple : (float, float, str, bool) 

so I can just do True to have something reasonably ok.